tests/layer_tests/mo_python_api_tests/test_mo_convert_import_from_memory.py

```diff
@@ -0,0 +1,475 @@
+# Copyright (C) 2018-2022 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+import openvino.runtime as ov
+import pytest
+import torch
+from openvino.runtime import PartialShape, Dimension, Model
+
+from common.mo_convert_test_class import CommonMOConvertTest
+
+
```

Comment:
I propose to split this suite to several files for each framework.

Reply:
Done.